syntax = "proto3";

package astre.ecs;

message ScriptComponent
{
  string name = 1;
}